docs/dev/api-reference/legacy.rst

```diff
@@ -4,6 +4,11 @@ Legacy API
 The "Legacy API" provides feature parity with `pypi-legacy`_, hence the term
 "legacy".
 
+.. note::
+  This API is available as both HTML and JSON as described in `PEP 691`_.
+
+  It is reccommended that new intergrations use the JSON version.
```

Comment:
Two types, should be:

```
It is recommended that new integrations use the JSON version.
```

I can make a PR if wanted.  Dinner first though. :-)

Reply:
https://github.com/pypi/warehouse/pull/16644